### PR TITLE
Adapt yast2_proxy to new startup widget

### PR DESCRIPTION
**Pull request by mloviska**
- Related ticket: [[fate#319428][functional][y] Adapt proxy yast module tests to new ServiceWidget](https://progress.opensuse.org/issues/39728)
- Needles: [osd](https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/912)
[o3](https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/415)
- Verification runs: 
[opensuse-Tumbleweed-DVD-x86_64-Build20180813-yast2_ncurses@64bit](http://dhcp128.suse.cz/tests/5777#)
[sle-15-SP1-Installer-DVD-x86_64-Build20.3-yast2_ncurses@64bit](http://dhcp128.suse.cz/tests/5776#)
[opensuse-15.1-DVD-x86_64-Build284.1-yast2_ncurses@64bit](http://dhcp128.suse.cz/tests/5773)
[sle-12-SP4-Server-DVD-x86_64-Build0341-yast2_ncurses@64bit](http://dhcp128.suse.cz/tests/5772)

### CODE REVIEW

**Comment by jknphy** [comment]
LGTM, except for the needles, but I guess we can recreate them.

**Comment by jknphy** [comment]
I am using the in mloviska fork to get the needles, although I found that http://dhcp128.suse.cz/tests/5776#step/yast2_proxy/12 is going to fast in my machine: http://dhcp87.suse.cz/tests/2144#step/yast2_proxy/14 so I creating one without selection of Start... I will continue checking if it fits all together with new needles:
https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/913
https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/417/files

**Comment by jknphy** [comment]
Finally I found them in another PR not linked: https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/413

File: tests/console/yast2_proxy.pm [hunk 1]
```diff
@@ -84,7 +84,21 @@ sub run {
...
+        assert_screen 'yast2_ncurses_service_start_widget';
+        send_key 'alt-f';
+        send_key_until_needlematch 'yast2_ncurses_service_start_widget_start_after_conf', 'up';
+        send_key 'ret';
+        assert_screen 'yast2_ncurses_service_start_widget_check_start_after_conf';
+        send_key 'alt-a';
+        send_key_until_needlematch 'yast2_ncurses_service_start_widget_start_on_boot', 'up';
+        send_key 'ret';
+        assert_screen 'yast2_ncurses_service_start_widget_check_start_on_boot';
+        '';
```
Comment (rwx788) [inline tests/console/yast2_proxy.pm]: I will remove this line here

**Pull request merged**